rmw_zenoh_cpp/src/detail/attachment_helpers.hpp

```diff
@@ -15,34 +15,37 @@
 #ifndef DETAIL__ATTACHMENT_HELPERS_HPP_
 #define DETAIL__ATTACHMENT_HELPERS_HPP_
 
-#include <zenoh.h>
+#include <cstdint>
+#include <vector>
+
+#include <zenoh.hxx>
 
 #include "rmw/types.h"
 
 namespace rmw_zenoh_cpp
 {
-///=============================================================================
 class AttachmentData final
 {
 public:
-  AttachmentData(
-    const int64_t sequence_number,
-    const int64_t source_timestamp,
-    const uint8_t source_gid[RMW_GID_STORAGE_SIZE]);
-  explicit AttachmentData(const z_loaned_bytes_t *);
+  explicit AttachmentData(
```

Comment:
We don't need to define this constructor as `explicit` as there are multiple arguments. 
```suggestion
 AttachmentData(
```